build/yamls/base/crds.yml

```diff
@@ -410,6 +414,10 @@ spec:
                                 cidr:
                                   type: string
                                   format: cidr
+                      destinationGroups:
+                        type: array
+                        items:
+                          type: string
```

Comment:
how do users refer to namespaced Groups in the future?

Reply:
i think the namespaced groups will be for ANP and not ACNP.. at least that is what i had in mind